anti_patterns/borrow_clone.md

```diff
@@ -0,0 +1,56 @@
+# Clone to satisfy the borrow checker
+
+## Description
+
+The borrow checker prevents rust users from developing otherwise unsafe code by
+ensuring that either: only one mutable reference exists, or potentially many but
+all immutable references exist. If the code written does not hold true to these 
+conditions, this anti-pattern arises when the developer resolves the compiler
+error by cloning the variable.
+
+
+## Example
+
+```rust fn main() {
```

Comment:
Also, I don't think you need to wrap the code in `fn main()` -- the other examples in this repo (along with the official docs) keep the code bare.